helix-term/src/ui/statusline.rs

```diff
@@ -426,6 +426,7 @@ where
     write(context, title, None);
 }
 
+// TODO: Make this customizable (will entail an update in the `editor.statusline` section of `book/src/configuration.md`).
```

Comment:
This should be a new issue or discussion instead of a TODO comment. I don't think this necessarily needs to be configurable. The discussion of that should be separate from this doc update though

Reply:
I updated the commit to just include the documentation update.